queue/abstract.lua

```diff
@@ -7,7 +7,7 @@ local qc       = require('queue.compat')
 local num_type = qc.num_type
 local str_type = qc.str_type
 
-local session = box.session
+local connection = box.session
```

Comment:
It looks as a 'connection' module that is a bit strange: we have no such server-side entity. How about defining a function to obtain a connection id, like so: `local connection_id = box.session.id`? Not sure here.

Reply:
Additionally I use `connection.on_disconnect()`